elyra/pipeline/airflow/package_catalog_connector/airflow-package-catalog.json

```diff
@@ -56,27 +56,29 @@
         },
         "airflow_package_download_url": {
           "title": "Airflow package download URL",
-          "description": "URL where the Apache Airflow package wheel can be downloaded.",
+          "description": "URL where the Apache Airflow package wheel can be downloaded",
           "type": "string",
           "uihints": {
             "category": "Source"
           }
         },
-        "auth_user": {
-          "title": "User",
-          "description": "Download user id",
+        "auth_id": {
+          "title": "Id",
+          "description": "Id that has read access for the specified URL resource",
           "type": "string",
           "uihints": {
-            "category": "Source"
+            "minLength": 1,
+            "category": "Source credentials"
           }
         },
         "auth_password": {
           "title": "Password",
-          "description": "Password for download user",
+          "description": "Password for specified id",
```

Comment:
Password or API key for specified id.

Overall, both in the popup hints and code and GUI, we might also think of calling it either user id or username.